index.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
```

Comment:
- [ ] [REQUIRED] Please, make sure that the mobile version is working properly. You have a huge scrollbar down there.

![image](https://user-images.githubusercontent.com/60979926/171964851-af9ac547-cdab-483a-b0b8-1c41ef731296.png)